core/helper.py

```diff
@@ -166,6 +166,15 @@ def print_err_response(err_response):
     print(LONG_LINE)
 
 
+def post(url_name, json=None, files=None):
+    host, cookies = get_node_creds(config)
+    url = construct_url(host, ROUTES[url_name])
+    response = post_request(url, json=json, files=files, cookies=cookies,)
```

Comment:
I think it is better to remove comma from "cookies=cookies,". Even it is possible according to python syntax it seems a bit confusing: "cookies," looks like tuple instead of string

Reply:
typo, I'll remove it